src/karmen_frontend/src/components/printers/printer-settings.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import PrinterEditForm from "../../components/forms/printer-edit-form";
+
+class PrinterSettingsForm extends React.Component {
```

Comment:
Please transform this into functional component, it's lot more error prone and descriptive. It's quite straightforward.